docs/diagrams/instance-state.svg

```diff
@@ -1,2 +1,2 @@
" y="596" width="120" height="40" rx="16" ry="16" fill="#ffffc0" stroke="#ff0000" pointer-events="none"/><g transform="translate(41.5,607.5)"><switch><foreignObject style="overflow:visible;" pointer-events="all" width="91" height="16" requiredFeatures="http://www.w3.org/TR/SVG11/feature#Extensibility"><div xmlns="http://www.w3.org/1999/xhtml" style="display: inline-block; font-size: 15px; font-family: Helvetica; color: rgb(0, 0, 0); line-height: 1.2; vertical-align: top; width: 92px; white-space: nowrap; word-wrap: normal; text-align: center;"><div xmlns="http://www.w3.org/1999/xhtml" style="display:inline-block;text-align:inherit;text-decoration:inherit;">Shuttingdown</div></div></foreignObject><text x="46" y="16" fill="#000000" text-anchor="middle" font-size="15px" font-family="Helvetica">Shuttingdown</text></switch></g><rect x="27" y="721" width="120" height="40" rx="16" ry="16" fill="#ffffc0" stroke="#ff0000" pointer-events="none"/><g transform="translate(49.5,732.5)"><switch><foreignObject style="overflow:visible;" pointer-events="all" width="74" height="16" requiredFeatures="http://www.w3.org/TR/SVG11/feature#Extensibility"><div xmlns="http://www.w3.org/1999/xhtml" style="display: inline-block; font-size: 15px; font-family: Helvetica; color: rgb(0, 0, 0); line-height: 1.2; vertical-align: top; width: 75px; white-space: nowrap; word-wrap: normal; text-align: center;"><div xmlns="http://www.w3.org/1999/xhtml" style="display:inline-block;text-align:inherit;text-decoration:inherit;">Terminated</div></div></foreignObject><text x="37" y="16" fill="#000000" text-anchor="middle" font-size="15px" font-family="Helvetica">Terminated</text></switch></g><rect x="230" y="721" width="120" height="40" rx="16" ry="16" fill="#ffffc0" stroke="#ff0000" pointer-events="none"/><g transform="translate(269.5,732.5)"><switch><foreignObject style="overflow:visible;" pointer-events="all" width="41" height="16" requiredFeatures="http://www.w3.org/TR/SVG11/feature#Extensibility"><div xmlns="http://www.w3.org/1999/xhtml" style="display: inline-block; font-size: 15px; font-family: Helvetica; color: rgb(0, 0, 0); line-height: 1.2; vertical-align: top; width: 42px; white-space: nowrap; word-wrap: normal; text-align: center;"><div xmlns="http://www.w3.org/1999/xhtml" style="display:inline-block;text-align:inherit;text-decoration:inherit;">Failed</div></div></foreignObject><text x="21" y="16" fill="#000000" text-anchor="middle" font-size="15px" font-family="Helvetica">Failed</text></switch></g><ellipse cx="87" cy="816" rx="11" ry="11" fill="#000000" stroke="#ff0000" pointer-events="none"/><ellipse cx="87" cy="816" rx="15" ry="15" fill="none" stroke="#ff0000" pointer-events="none"/><ellipse cx="290" cy="816" rx="11" ry="11" fill="#000000" stroke="#ff0000" pointer-events="none"/><ellipse cx="290" cy="816" rx="15" ry="15" fill="none" stroke="#ff0000" pointer-events="none"/><ellipse cx="87" cy="36" rx="11" ry="11" fill="#000000" stroke="#ff0000" pointer-events="none"/></g></svg>
```

Comment:
You should use xmllint to turn this in a multiline xml file. Everything on one line is great for browsers but not for git. It would cause every change to the file to replace the entire file.